### DISCUSSION TITLE
mHM v6.0 release

### DISCUSSION THREAD

**Discussion by StephanThober** [mhm-ufz/mHM#9] [Show and tell]
Hi all,

we want to inform the community that we are currently heavily working on the mHM v6.0 release that is scheduled for release this summer. The most important change is that the mHM v6.0 release will use the new implementation of MPR that is more flexible than the one of previous releases. This new MPR version can be found here: https://git.ufz.de/chs/MPR The advantage is that transfer function and upscaling operators can be changed easily in a text file. 

Further major changes are:
- all input files will be in netCDF format that includes morphological files, currently stored as ascii files
- drop support for binary files

We will provide python scripts to convert ascii files to netCDF files to ease the transition to v6.0.

Please let us know if this works for you or whether these changes would break too many workflows.

Best,
Sebastian & Stephan

**Comment by cuneyd** [comment]
Hey, 
I wonder when will the users be able to use hourly inputs and outputs? After v6 this year?

**Comment by StephanThober** [comment]
Hey Cuneyd,

we will most likely release a v5 version with the feature to read hourly inputs in March/April, so this will be available before the v6 release.

Best,
Stephan
